### PR TITLE
404 lists CHEF-KOCH (COCK)

### PR DESCRIPTION
**Pull request by bogachenko**
12345

### CODE REVIEW

File: data/FilterList.json [hunk 1]
```diff
@@ -5504,21 +5504,6 @@
...
-    "donateUrl": "https://github.com/CHEF-KOCH/Donations",
-    "emailAddress": "CHEF-KOCH@protonmail.com",
-    "homeUrl": "https://github.com/CHEF-KOCH/CKs-FilterList",
-    "issuesUrl": "https://github.com/CHEF-KOCH/CKs-FilterList/issues",
-    "licenseId": 17,
-    "name": "CHEF-KOCH's NSABlocklist FilterList",
-    "publishedDate": "2018-05-17T15:44:44",
-    "syntaxId": 1,
-    "updatedDate": "2018-12-30T15:04:23",
-    "viewUrl": "https://raw.githubusercontent.com/CHEF-KOCH/CKs-FilterList/master/filters/nsablocklist.txt"
```
Comment (collinbarrett) [inline data/FilterList.json]: This list seems to still exist, but has been moved to: https://raw.githubusercontent.com/CHEF-KOCH/CKs-FilterList/master/Anti-Corp/filters/nsablocklist.txt

I suspect it is similar with the others? I'd rather wait until someone has the time to fix the broken urls in a PR than remove these all entirely.

Thanks.

Comment (bogachenko) [reply data/FilterList.json]: fuck. mistake

Comment (collinbarrett) [reply data/FilterList.json]: no worries. thanks for contributing!

**Comment by bogachenko** [comment]
@collinbarrett fixed all the links, you can check... can that the broken or forgot... and Yes CHIEF KOCH fucked me move their lists

**Pull request merged**